egs/wsj/s5/steps/limit_num_gpus.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# This script functions as a wrapper of a bash command that uses GPUs.
```

Comment:
this should be in utils/.  
Actually, create a new subdirectory utils/parallel/, add this script to there, move pbs.pl, run.pl, slurm.pl and queue.pl there and create soft links from utils for those existing scripts for back-compatibility.


(the norm for scripts that don't take the